designer/server/src/main/resources/db/migration/hsql/V1_052__AddLatestActionAndLatestVersion.sql

```diff
@@ -0,0 +1,47 @@
+ALTER TABLE "processes"
+    ADD COLUMN "latest_version_id" NUMBER;
+ALTER TABLE "processes"
+    ADD COLUMN "latest_finished_action_id" UUID;
+ALTER TABLE "processes"
+    ADD COLUMN "latest_finished_cancel_action_id" UUID;
+ALTER TABLE "processes"
+    ADD COLUMN "latest_finished_deploy_action_id" UUID;
+
+-- TODO this update could be much more optimized
```

Comment:
apparently this very straightforward select is quite optimized, at least from my testing with postgres, a more involved update with CTEs performed far worse